java/ql/test/query-tests/ContinueInFalseLoop/A.java

```diff
@@ -32,14 +32,14 @@ void test1(int x, Cond c) {
 
     // --- while, for loops ---
 
-    while (false) {
+    while (c.cond()) {
       if (c.cond())
         continue; // GOOD [never reached, if the condition changed so it was then the result would no longer apply]
       if (c.cond())
         break;
     }
 
-    for (i = 0; false; i++) {
+    for (i = 0; c.cond(); i++) {
```

Comment:
These two test cases no longer make sense, so it's confusing to leave them like this. Best to just delete them.